src/System.Management.Automation/AssemblyInfo.cs

```diff
@@ -2,9 +2,6 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyFileVersionAttribute("3.0.0.0")]
-[assembly: AssemblyVersion("3.0.0.0")]
```

Comment:
Change the assembly version from 3.0.0.0 to 6.0.0.0 will break all existing assemblies that reference to S.M.A 3.0.0.0, for example the existing PakcageManagement module.